### ISSUE TITLE
Error message in case of `not a git repository` error is far too verbose

### ISSUE THREAD

**Issue by PawelLipski** [VirtusLab/git-machete#171]
```
$ git machete 
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/git_machete-3.3.1-py3.7.egg/git_machete/git_operations.py", line 116, in get_git_dir
  File "/usr/local/lib/python3.7/site-packages/git_machete-3.3.1-py3.7.egg/git_machete/git_operations.py", line 52, in popen_git
git_machete.exceptions.MacheteException: git rev-parse --git-dir returned 128

stderr:
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/git-machete", line 4, in <module>
    __import__('pkg_resources').run_script('git-machete==3.3.1', 'git-machete')
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 666, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1469, in run_script
    exec(script_code, namespace, namespace)
  File "/usr/local/lib/python3.7/site-packages/git_machete-3.3.1-py3.7.egg/EGG-INFO/scripts/git-machete", line 4, in <module>
    __import__('pkg_resources').run_script('git-machete==3.3.1', 'git-machete')
  File "/usr/local/lib/python3.7/site-packages/git_machete-3.3.1-py3.7.egg/git_machete/cmd.py", line 1968, in main
  File "/usr/local/lib/python3.7/site-packages/git_machete-3.3.1-py3.7.egg/git_machete/cmd.py", line 1975, in launch
  File "/usr/local/lib/python3.7/site-packages/git_machete-3.3.1-py3.7.egg/git_machete/cmd.py", line 70, in __init__
    doc_header = "Documented commands (type help <topic>):"
  File "/usr/local/lib/python3.7/site-packages/git_machete-3.3.1-py3.7.egg/git_machete/git_operations.py", line 122, in get_git_subpath
  File "/usr/local/lib/python3.7/site-packages/git_machete-3.3.1-py3.7.egg/git_machete/git_operations.py", line 118, in get_git_dir
git_machete.exceptions.MacheteException: Not a git repository
```

**Comment by rifel123** [comment]
After applying verbose flag to MacheteException, stack trace looks now:
<img width="575" alt="Screenshot 2021-08-18 at 08 59 05" src="https://user-images.githubusercontent.com/73847352/130077257-e12100c4-b09b-4d5f-bd21-7dc325d02a7e.png">

Note: verbose flag has to be set to False in exception which breaks flow of program to take effect.

**Comment by PawelLipski** [comment]
Is just `develop` affected? or `master` = v3.3.0 as well?

**Comment by rifel123** [comment]
actually, i was able to reproduce this trace only when calling git machete status directly from python. Otherwise i got always pretty short message:
<img width="330" alt="Screenshot 2021-08-20 at 08 26 50" src="https://user-images.githubusercontent.com/73847352/130189798-779d80e6-602f-4fb7-8276-f0bd27de4c2f.png">

**Comment by rifel123** [comment]
fixed in #179
